index.html

```diff
@@ -4,9 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Hosting Stron - Nieograniczony Hosting i Darmowa Domena -</title>
+    <link rel="stylesheet" href="./css/main.css">
 </head>
+
+
+
 <body>
-    
+<header class="support">
+    <div class="support__top">
+        <h1 class="support__h1">Wspieramy Ukrainę</h1>
+        <img class="support__img" src="Images/ukraine.webp" alt="ukraine">
+    </div>
+
+    <div class="support__text">
+        <p>Atak rządu rosyjskiego na Ukrainę naraził na niebezpieczeństwo miliony niewinnych osób. Wspieramy Ukrainę po to, by pomóc jej odzyskać wolność i aby 
+            bronić demokracji. Jeśli chcesz wesprzeć Ukrainę i jej obywateli w tym trudnym czasie, kliknij <a href="http://google.pl">tutaj, aby przekazać darowiznę.</a></p>
+            
+
+    </div>
+</header>
+
```

Comment:
Co to za puste linijki? I dodatkowo masz tu jakis dziwny format. Uzyc vs code do formatowania kodu